### PR TITLE
[DOCFIX] Update rocksdb metastore doc

### PR DESCRIPTION
**Pull request by yuzhu**
### What changes are proposed in this pull request?

Update docs for rocksdb default size.

### Why are the changes needed?
Recent change in default onheap cache for rocksdb metastore. 

### Does this PR introduce any user facing changes?

DocFix

### CODE REVIEW

**Review by sg-c — COMMENTED** [review]
Can you also update "The default cache size is 10 million inodes, which comes out to around 10GB of memory." to something like "By default, (JVM-memroy-size / 4KB) inodes will be cached in memory. For example, 10GB 
of available memory will cache around 2.5 million inodes."

**Review by ggezer — COMMENTED** [review]
Could you explicitly mention we set this dynamically based on configured heap size?

File: docs/en/operation/Metastore.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ The default metastore is the `ROCKS` metastore.
 The ROCKS metastore uses a mixture of memory and disk. Metadata eventually gets
 written to disk, but a large, in-memory cache stores recently-accessed metadata. The
 cache buffers writes, asynchronously flushing them to RocksDB as the cache fills up.
-The default cache size is 10 million inodes, which comes out to around 10GB of memory.
+The default cache size is dynamically set to be (JVM Heap size / 4KB).
+In the default configuration of 8GB heap size, we will cache 2 million inodes.
```
Comment (ggezer) [inline docs/en/operation/Metastore.md:26]:
```suggestion
In the default configuration of 8GB heap size, Alluxio master will cache 2 million inodes.
```

File: docs/en/operation/Metastore.md [hunk 2]
```diff
@@ -37,9 +38,10 @@ alluxio.master.metastore=ROCKS
 
 * `alluxio.master.metastore.dir`: A local directory for writing RocksDB data.
 Default: `${work_dir}/metastore`
-* `alluxio.master.metastore.inode.cache.max.size`: A hard limit on the size of the inode cache.
-Increase this to improve performance if you have spare master memory. Every million inodes
-takes roughly 1GB of memory. Default: `10000000` (10 million)
+* `alluxio.master.metastore.inode.cache.max.size`: A hard limit on the number of entries in the on-heap inode cache.
+Increase this to improve performance if you have spare master memory. 
+The default value for this configuration is dynamically set to be (JVM Heap size / 4KB).
+For example, when xmx setting is set to the default 8GB, we will cache 2 million inodes.
```
Comment (ggezer) [inline docs/en/operation/Metastore.md:44]:
```suggestion
For example, when xmx setting is set to the default 8GB, Alluxio master will cache 2 million inodes.
```

**Review by ggezer — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**